client/views/courses/events/course.events.js

```diff
@@ -4,16 +4,24 @@ Template.courseEvents.onCreated(function() {
 
 	instance.eventSub = subs.subscribe('eventsForCourse', courseId);
 
+	instance.showAllEvents = new ReactiveVar(false);
+
 	instance.haveEvents = function() {
 		return eventsFind({ course: courseId, start: minuteTime.get() }).count() > 0;
 	};
 
+	instance.haveMoreEvents = function() {
+		return eventsFind({ course: courseId, start: minuteTime.get() }).count() > 4;
```

Comment:
The literal 4 here is called a "magic number" because we don't know what it means. That style deficiency was already present before you started so I'm to blame here :-) The value is repeated in the futureEvents helper below which makes it worse because we'd have to remember changing it in both places.

It would be better for robustness  the understanding of the code if this was defined at the start of `onCreated` with `var maxEventsShown = 4`.

Maybe the code would be even clearer by directly using in all places `instance.maxEventsShown = new ReactiveVar(4);` and then set that to zero when the user clicks on "show all".